### PR TITLE
Remove unused code in MySQLData

### PR DESCRIPTION
**Pull request by 0xTim**

### CODE REVIEW

**Comment by tanner0101** [comment]
Was this bug fixed on Linux?

**Comment by 0xTim** [comment]
@tanner0101 `nanoSecond` (that was removed) was never actually used in that scope so it was redundant

File: Sources/MySQL/Connection/MySQLData.swift [hunk 1]
```diff
@@ -583,40 +583,33 @@ extension Date {
...
-        #endif
-        
+
         comps.value.year = numericCast(time.year)
         comps.value.month = numericCast(time.month)
         comps.value.day = numericCast(time.day)
         comps.value.hour = numericCast(time.hour)
         comps.value.minute = numericCast(time.minute)
         comps.value.second = numericCast(time.second)
         comps.value.nanosecond = numericCast(time.microsecond) * 1_000
```
Comment (eneko) [inline Sources/MySQL/Connection/MySQLData.swift]: Should this line be removed too?

Comment (0xTim) [reply Sources/MySQL/Connection/MySQLData.swift]: Meh space for clarity is never a bad thing!

**Pull request closed without merging**